vulcanai/models/basenetwork.py

```diff
@@ -494,6 +516,61 @@ def _assert_same_devices(self, comparison_device=None):
                     comparison_device,
                     incompatible_collector))
 
+    # modified from
+    # https://github.com/Bjarten/early-stopping-pytorch/blob/master
+    # /pytorchtools.py
+    class EarlyStopping:
+        """Early stops the training if validation loss doesn't improve after
```

Comment:
would there be a use case to use the validation accuracy as the stopping metric instead of the loss? just seeing if there's a way to generalize this metric unless this is the standard way of identifying early stopping procedures

Reply:
I haven't a clue what is standard. I can make it a parameter, though, that's probably worthwhile yes?

Reply:
Yeah having it parametrized is ideal